### PR TITLE
Refactor request token link generation

### LINKED ISSUE #5481: Change deep linking to use the https protocol
### Description

Change deeplink `lisk:` protocol trigger to https protocol

### Acceptance criteria

- Change the deep link trigger protocol from using `lisk:` using the lisk domain i.e `https://lisk.com`.
- Refactor redirect logic to open up send modal when a `/send` path is provided against the new domain.

### Motivation

- As part of security audit the trail of bits recommended to move from launch protocol to https for more secure and trusted domain.

### PR DESCRIPTION
**Pull request by eniolam1000752**
### What was the problem?

This PR resolves #5481

### How was it solved?

- [x] Change link generated on request token from using the `lisk:` domain to the lisk `https:` domain.
- [x] Fix failing tests

### How was it tested?

- Request token
- `https://lisk.com/send?<reset of the required query params>` should be copied to the clip board.

### CODE REVIEW

File: setup/jest.config.js [hunk 1]
```diff
@@ -346,6 +346,7 @@ module.exports = {
     VERSION: '',
     REACT_APP_DEFAULT_NETWORK: 'undefined',
     LISK_ENABLE_DEV_TOOL: false,
+    LISK_DOMAIN: 'https://lisk-test.com',
```
Comment (oskarleonard) [inline setup/jest.config.js]: dont we need `/send` here as well. Or why is this CONST 3 different things?
` "LISK_DOMAIN": true,` // Boolean
`LISK_DOMAIN: '"https://lisk.com/send"',`  // has `/send`
`LISK_DOMAIN: 'https://lisk-test.com',`

Comment (ManuGowda) [reply setup/jest.config.js]: Now the lisk.com deployment is updated, @eniolam1000752 will update this and test

File: setup/config/webpack.config.react.js [hunk 2]
```diff
@@ -103,6 +103,7 @@ const config = {
     }),
     new DefinePlugin({
       VERSION: `"${bundleVersion}"`,
+      LISK_DOMAIN: '"https://staging.lisk.com"',
```
Comment (ManuGowda) [inline setup/config/webpack.config.react.js]:
```suggestion
      LISK_DOMAIN: '"https://lisk.com"',
```

File: setup/jest.config.js [hunk 3]
```diff
@@ -346,6 +346,7 @@ module.exports = {
     VERSION: '',
     REACT_APP_DEFAULT_NETWORK: 'undefined',
     LISK_ENABLE_DEV_TOOL: false,
+    LISK_DOMAIN: 'https://staging.lisk.com',
```
Comment (ManuGowda) [inline setup/jest.config.js]:
```suggestion
    LISK_DOMAIN: 'https://lisk.com',
```

**Review by ManuGowda — APPROVED** [review]

**Comment by ManuGowda** [comment]
Works as expected.

**Pull request merged**